test/compile/selection/nearest.test.ts

```diff
@@ -21,7 +21,7 @@ function getModel(markType: any) {
   model.component.selection = selection.parseUnitSelection(model, {
     "one": {"type": "single", "nearest": true},
     "two": {"type": "multi", "nearest": true},
-    "three": {"type": "interval", "nearest": true},
+    "three": {"type": "interval"},
```

Comment:
Why are these get removed? We will need input from @arvind if removing these make sense.

Reply:
I’m guessing these were removed because they are actually not valid Vega-Lite statements. I just had them here to verify that we did the right thing (eg ignore or throw errors as appropriate) if a user specified them. Will go through the other selection test changes too and comment if something looks amiss.

Reply:
And now the typescript compiler is smart enough to notice.